api/mess/models.py

```diff
@@ -40,18 +40,13 @@ class MessMenu(models.Model):
     items = models.ManyToManyField(MenuItem)
     weekdays = models.ManyToManyField(WeekDay)
 
-class MessMenu(models.Model):
-    slot = models.ForeignKey(MenuSlot, on_delete=models.CASCADE)
-    items = models.ManyToManyField(MenuItem)
-    weekdays = models.ManyToManyField(WeekDay)
-
 # data base model for feedback
 # status is for whether the feedback is viewed by mess manager or not.
 class MessFeedback(models.Model):
     user = models.ForeignKey(settings.AUTH_USER_MODEL,
                              on_delete=models.CASCADE)
     body = models.TextField(editable=True, null=False, blank=True)
-    mess_type = models.CharField(max_length=50)
+    mess_meal = models.CharField(max_length=50)
```

Comment:
Why is this required? Have you made sure that this won't break other things?

Reply:
yes this is necessary as per the feedback designed for the mess

Reply:
Won't the frontend model also change? Changing this only in the backend will cause errors.

Reply:
no changes are required at the front end as payload names for the API are the same only the model field is changed. if any changes are required then are requested to be made.